rocket.py

```diff
@@ -0,0 +1,33 @@
+import pygame
+import os 
+from menu import *
+#First let's make a moving rectangle here and then modify to make a moving rocket
+class Rocket():
+    def __init__(self, game):
+        self.game = game #Use a game object
+        self.width, self.height = self.game.DISPLAY_W / 20, self.game.DISPLAY_H / 20#Size of the rectangle
+        #Initial position of the rectangle (bottom, centered)
+        self.x, self.y = self.game.DISPLAY_W / 2 - self.width/2, self.game.DISPLAY_H - self.height
+        self.vel=1 #Moving velocity
+        self.spaceship = pygame.transform.scale(pygame.image.load(os.path.join("assets/sprites", "spaceship-a.svg")), (round(self.width),round(self.height)))
+
+
+    def blit_rocket(self):
+        self.game.display.blit(self.spaceship, (self.x,self.y))
+        #pygame.draw.rect(self.game.display,(255,0,0),(self.x,self.y,self.width,self.height))
```

Comment:
Is this meant to stay?

Reply:
the commented out rectangle no
the function though, yeah. I thought it was cleaner that way